client/src/main/java/edu/colostate/cs/cs414/chesshireCoders/jungleClient/view/impl/HomeViewImpl.java

```diff
@@ -104,6 +106,11 @@ private void playClicked() {
             showError(e.getMessage());
         }
     }
+    
+    private void colorblindClicked() {
+    	colorblind = !colorblind;
+    	reloadActiveGame();
```

Comment:
You should check that there is an active game before trying to change it (NullPointerException).

Reply:
Wow can't believe I missed that. Thanks.

Reply:
Nice catch. Fixed.